src/commands/edge/channels/unassign.ts

```diff
@@ -1,10 +1,44 @@
 import { flags } from '@oclif/command'
 
+import { ChooseOptions, chooseOptionsWithDefaults, selectFromList, stringTranslateToId } from '@smartthings/cli-lib'
+
 import { EdgeCommand } from '../../../lib/edge-command'
+import { DriverChannelDetails } from '../../../lib/endpoints/channels'
 import { chooseChannel } from '../channels'
-import { chooseDriver } from '../drivers'
 
 
+export interface NamedDriverChannelDetails extends DriverChannelDetails {
+	name: string
+}
```

Comment:
Does this make more sense to live in `../../../lib/endpoints/channels`?

Reply:
In my opinion, no. It's specifically for the `chooseAssignedDriver` method below. Hopefully at some point in the future, we can get them to add name to the endpoint and we can just add `name` directly to `DriverChannelDetails`.